Formula/breakpad.rb

```diff
@@ -0,0 +1,69 @@
+class Breakpad < Formula
+  desc "Components implement a crash-reporting system."
+  homepage "https://chromium.googlesource.com/breakpad/breakpad/"
+
+  url "https://chromium.googlesource.com/breakpad/breakpad",
+      :using => :git,
+      :revision => "67d5567177301d0c24303f26ad119ab7bd7fab40"
+  version "0.1.0"
```

Comment:
just make `brew audit` happy, we may change it to 1.0.0 as default version of Breakpad.Framework